bootstrap/src/main/java/io/lumigo/javaagent/LumigoAgent.java

```diff
@@ -33,8 +35,14 @@ public static void agentmain(final String agentArgs, final Instrumentation inst)
       System.setProperty("lumigo.debug", "true");
     }
     if (is_switch_off()) {
+      System.err.println(
+          "Lumigo OpenTelemetry JavaAgent distribution disabled via the 'LUMIGO_SWITCH_OFF' environment variable");
       return;
     }
+    System.out.println(
+        "Loading the Lumigo OpenTelemetry JavaAgent distribution (version "
+            + parseVersion()
+            + ") (injection mode: automatic injection)");
```

Comment:
In Java there is only automatic injection (as opposed to Node.js and Python where the distros can also be used as libraries). We can spare the injection mode part of the message I think.